bundler.go

```diff
@@ -607,8 +613,18 @@ func (b *Bundler) finishDarwin(environmentPath, binaryPath string) (err error) {
 		return
 	}
 
-	// Move binary
 	var macOSBinaryPath = filepath.Join(macOSPath, b.appName)
+
+	var infoPlist *plister.InfoPlist
+	if b.infoPlist != nil {
+		infoPlist = plister.MapToInfoPlist(b.infoPlist)
+		binaryName, _ := infoPlist.Get("CFBundleExecutable").(string)
```

Comment:
Could you still check the type assertion is successful ?

```go
if binaryName, ok := infoPlist.Get("CFBundleExecutable").(string); ok && len(binaryName) > 0 {
  macOSBinaryPath = filepath.Join(macOSPath, binaryName)
}
```

Reply:
In this case, we do not need to do redundant checks. It is enough for us to verify that the result of the casting is not equal to the empty string. Look at the example how it works https://goplay.space/#3LZLCttXr7U

One more thing. In the idiomatic Go, checking for empty strings should be done by comparing with an empty string, and not by comparing its length (https://dmitri.shuralyov.com/idiomatic-go#empty-string-check)